models/opencontrol.go

```diff
@@ -83,7 +83,7 @@ func (openControl *OpenControl) LoadComponents(directory string) []error {
 	var wg sync.WaitGroup
 	componentsDir, err := ioutil.ReadDir(directory)
 	if err != nil {
-		return []error{ErrReadDir}
+		return []error{errors.New("Error: Unable to read the directory "+directory)}
```

Comment:
I forgot about this while I was writing tests. It works with the string, now, but now that I have tests working we could go this route too.